examples/lang/hetzner0.mcl

```diff
@@ -0,0 +1,40 @@
+import "os"
+#import "deploy"
+
+# read token from a local path outside repo:
+$f = "../mgmt-dev/apitoken.txt"
+$token = os.readfile($f)
+
+# alternatively, read from the deploy:
+# $token = deploy.readfile(...)
+
+# or give the string directly:
+# $token = "..."
+
+hetzner "resourcedemo" {
```

Comment:
Would it make sense to use a namespaced name for this resource, like `hetzner:vm` (or whatever nomenclature Hetzner uses for their instances) in case you/someone wants to add other Hetzner resources in the future, to say add SSH keys or other things that Hetzner provide

Reply:
Agreed, having something like hetzner:vm or hetzner:server would be great. We've already been thinking about a dedidated hetzner resource for ssh keys, although I haven't gotten to that implementation yet. 

What would be the appropriate naming structure for mcl lang then? I'm not clear yet on how that is generated.

Reply:
Done. I named it hetzner:server since the hcloud-go package uses server terminology rather than VMs, but we can change it if you prefer something else. The filename is now `hetzner_server.go` and the resource is `HetznerServerRes`.

Reply:
See other comment from @dantefromhell ... hetzner:vm sounds sensible, but so does hetzner:server ... As long as you've addressed the comment about what the physical server variant would be called (just suggest a name so we know our current choice is hopefully future proof) then I'm fine with that.

Remember the goals in order:

1) It should make the most sense to _our_ users.
2) It should make sense to the hetzner API authors and internal hetzner folks. We'd like to keep things consistent in their world and use their naming conventions, but if that would be in conflict with what makes sense here, do what's best for mgmt users.